controllers/storagecluster/cephcluster.go

```diff
@@ -1564,37 +1560,6 @@ func generateCephReplicatedSpec(initData *ocsv1.StorageCluster, poolType string)
 	return crs
 }
 
-// setMonTargetPgPerOsd sets the mon_target_pg_per_osd value to 400 if not already set
-func setMonTargetPgPerOsd(cephConfig *map[string]map[string]string) {
-	if *cephConfig == nil {
-		*cephConfig = make(map[string]map[string]string)
-	}
-	if global, exists := (*cephConfig)["global"]; exists {
-		if _, exists := global["mon_target_pg_per_osd"]; !exists {
-			global["mon_target_pg_per_osd"] = "400"
```

Comment:
Where is this set now? I only see deleted lines in the PR, so maybe there is another place it's set by default that just isn't in the PR?

Reply:
Thanks for pointing it out, I had prepared another commit which I forgot to push. Now updated.